### PR TITLE
Refresh Recipe - v0.1.0, Never Published

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by rxm7706** [comment]
@conda-forge-admin, please rerender

**Comment by rxm7706** [comment]
@conda-forge/core @karthick1304  
This feedstock has been unmaintained for over two years. 
Can you review and merge, I have added myself as as a maintainer - I am happy to help keep this feedstock updated.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,39 +2,49 @@
...
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dpd_components-{{ version }}.tar.gz
   sha256: 613a6b17d3d7dd449be060e739e4ce36692b46fa012c3a86ee947f6337d09548
 
 build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```
Comment (hmaarrfk) [inline recipe/meta.yaml:15]: This should be increased

Comment (rxm7706) [reply recipe/meta.yaml:15]: @hmaarrfk 
I left the build number unchanged - as there have been no successful builds / published versions of this feedstock. 
I believe you are suing - Even if a feedstock has never successfully built or published - I should bump the build number.. 
https://anaconda.org/search?q=dpd_components 
LMK what the right course is. 
Thank you & Regards.

**Pull request merged**